Testing/SystemTests/tests/framework/SaveLoadNexusProcessed.py

```diff
@@ -109,3 +108,15 @@ def savedFilename(self):
 
     def validate(self):
         return self.compareWorkspaces()
+
+
+class SaveLoadNexusProcessedGroupingWorkspaceTest(SaveLoadNexusProcessedTestBase):
+    def createTestWorkspace(self):
+        LoadEmptyInstrument(InstrumentName="SNAP", OutputWorkspace="test_in")
+        CreateGroupingWorkspace(InputWorkspace="test_in", GroupDetectorsBy="Column", OutputWorkspace=self.test_ws_name)
+
+    def savedFilename(self):
+        return "tmp_saveload_nexusprocessed_emptygroup"
+
+    def validate(self):
```

Comment:
You don't need to override the base class implementation with what it already does. This method can be deleted.

Reply:
Fixed, also removed from other tests in file